README.md

```diff
@@ -63,47 +60,88 @@ security = {
 
 # request data for Questions API
 request = {
-	"type": "local_practice", "state": "initial",
-	"questions": [
-		{
-			"response_id": "60005",
-			"type": "association",
-			"stimulus": "Match the cities to the parent nation",
-			"stimulus_list": [
-				"London", "Dublin", "Paris", "Sydney"
-			],
-			"possible_responses": [
-				"Australia", "France",
-				"Ireland", "England"
-			],
-			"validation": {
-				"valid_responses": [
-					["England"], ["Ireland"], ["France"], ["Australia"]
-				]
-			}
-		}
-	]
+    "type": "local_practice", "state": "initial",
+    "questions": [
+        {
+            "response_id": "60005",
+            "type": "association",
+            "stimulus": "Match the cities to the parent nation",
+            "stimulus_list": [
+                "London", "Dublin", "Paris", "Sydney"
+            ],
+            "possible_responses": [
+                "Australia", "France",
+                "Ireland", "England"
+            ],
+            "validation": {
+                "valid_responses": [
+                    ["England"], ["Ireland"], ["France"], ["Australia"]
+                ]
+            }
+        }
+    ]
 }
 
 init = learnosity_sdk.request.Init(
-	'questions', security, secret,
-	request=request
+    'questions', security, secret,
+    request=request
 )
 
 # Get the JSON that can be rendered into the page and passed to LearnosityApp.init
 request = init.generate()
 ```
 
+Data API example:
+
+```python
+import learnosity_sdk.request
+
+endpoint = 'https://data.vg.learnosity.com/latest/itembank/items'
+
+security = {
+    'consumer_key': 'MY_API_KEY',
```

Comment:
Given that a real request is never made, I don't think we need to embed the demo keys into here.

Reply:
I can see that we do that in the PHP SDK documentation (although we don't use the consumer secret there). I don't know if we should because we don't want anyone to get the impression that the demos consumer key is some kind of magic number they need to pass (instead of their own consumer key).

Reply:
I think it's easier to get people started if they have something they can just copy and paste.

That said, and in light of some other comments below, yes, we don't want people to think it's a magic value, nor to get that hardcoded and checked-in in their SCM like -erm- some people did in the past 🏃 

I've taken to adding comments to that effect whenever I embed the demo key. See for example...

Well I promise you I did it somewhere, but not in the first place I looked... So I'm fixing that now q:

There:
https://github.com/Learnosity/learnosity-sdk-ruby/blame/master/README.md#L65